src/components/header/Header.css

```diff
@@ -0,0 +1,164 @@
+/* Removing Navbar on mobile */
+.navBar {
+  display: none;
+}
+
+/* Burger menu module style */
+.bm-burger-button {
+  position: fixed;
+  width: 36px;
+  height: 30px;
+  left: 36px;
+  top: 36px;
+}
+
+.menu {
+  height: 10vh;
+  position: sticky;
+  top: 0;
+  z-index: 1000;
+  margin-bottom: 1vh;
+  width: 100%;
+  background-color: #e5e5f7;
+  background-image:
+    linear-gradient(
+      30deg,
+      #f79a22 40%,
+      transparent 12.5%,
+      transparent 87%,
+      #f7bd51 87.5%,
+      #f1bc47
+    ),
+    linear-gradient(
+      150deg,
+      #facc4d 50%,
+      transparent 40%,
+      transparent 87%,
+      #f5a81b 87.5%,
+      #f1bc28
+    ),
+    linear-gradient(
+      30deg,
+      #facc4d 50%,
+      transparent 40%,
+      transparent 87%,
+      #f5a81b 87.5%,
+      #f1bc28
+    ),
+    linear-gradient(
+      50deg,
+      #facc4d 30%,
+      transparent 40%,
+      transparent 87%,
+      #f5a81b 87.5%,
+      #f1bc28
+    ),
+    linear-gradient(
+      60deg,
+      #facc4d 30%,
+      transparent 40%,
+      transparent 75%,
+      #f5a81b 75%,
+      #f1bc28
+    ),
+    linear-gradient(
+      60deg,
+      #f7d067 40%,
+      transparent 25.5%,
+      transparent 75%,
+      #f5a81b 75%,
+      #f1bc28
+    );
```

Comment:
Are you applying all of them on the same element ? Is every one ofthem bypassing the previous one ?